.github/workflows/docker.yml

```diff
@@ -67,43 +72,3 @@ jobs:
             - "WITH_LEDGER=false"
             - "BUILD_TAGS=muslc musl dynamic"
           skip: ${{ github.event_name == 'pull_request' }}
-
-  docker:
```

Comment:
This wasn't building so rather than debugging I just removed it. The heighliner image is what we run networks in our cluster with.